core/services/headtracker/head_broadcaster.go

```diff
@@ -68,23 +63,39 @@ func (hr *HeadBroadcaster) Start() error {
 
 func (hr *HeadBroadcaster) Close() error {
 	return hr.StopOnce("HeadBroadcaster", func() error {
+
+		hr.mutex.Lock()
+		hr.callbacks = make(callbackSet)
```

Comment:
What does this do?

Reply:
it clears the map - i added a comment now

Reply:
I mean, why is it necessary? We are closing it anyway?

Reply:
ah i had an impression it might be possible to start and stop it multiple times in the tests, but no, it can be stopped only once. Removed this now, thanks.